src/api/dependenciesConfig.js

```diff
@@ -133,6 +136,16 @@ export function ensureIdentityExists(
       if (dep.autoActivateOnIdentityCreation) {
         ledger.activate(id);
       }
+      if (!dep.periods.length) {
+        // Set default start period to one week from adding the dependency
+        return {
+          ...dep,
+          id,
+          periods: [
+            {"startTime": toISO(Date.now() + ONE_WEEK_IN_MS), "weight": 0.05},
```

Comment:
maybe just use Date.now? more robust to the change where we make the period 1 day instead of 1 week.
(neutral, could go either way)